example/deploy/rbac.yaml

```diff
@@ -33,14 +33,15 @@ rules:
 
 ---
 
-kind: RoleBinding
+kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: default-account-app-operator
 subjects:
 - kind: ServiceAccount
   name: default
+  namespace: default
```

Comment:
I don't actually understand why it made me provide a namespace here, so perhaps someone can sanity-check that for me.

Reply:
It needs to know the namespace for the service account all good